Mathlib/Topology/Algebra/InfiniteSum/Defs.lean

```diff
@@ -39,11 +56,16 @@ open scoped BigOperators Topology
 
 variable {α β γ : Type*}
 
-section HasSum
+section HasProd
+
+variable [CommMonoid α] [TopologicalSpace α]
 
-variable [AddCommMonoid α] [TopologicalSpace α]
+/-- Infinite product on a topological monoid
 
```

Comment:
Could you explain in the docstring what this actually is, in addition to referring to `hasSum`? In a tooltip, the user will not have the opportunity to see the definition of `hasSum`, so docstrings should be self-contained when possible.